### ISSUE TITLE
PSO2NGS (Japanese version) unstable behavior with XInput

### ISSUE THREAD

**Issue by velocity7** [Ryochan7/DS4Windows#2388]
**Describe the bug**
When DS4Windows is set to X360 mode with a DualSense on Windows startup, PSO2NGS responds properly when in XInput mode. However, if DS4Windows disconnects and reconnects the controller, PSO2NGS in XInput mode only responds to DS4Windows up until the title screen. 

In this state, PSO2NGS will only respond in DirectInput mode, but a regular X360 controller will still work with PSO2NGS in XInput. Restarting DS4Windows does not resolve the problem. A full PC restart temporarily resolves the issue.

**To Reproduce**
1. Connect a DualSense to DS4Windows
2. Set DS4Windows to X360 mode
3. Start PSO2NGS (JP) and switch to XInput mode
4. Confirm buttons work
5. Stop and start DS4Windows, or disconnect/reconnect controller

**Expected behavior**
All buttons should be responding correctly.

**Screenshots and Logs**
If applicable, add screenshots to help explain your problem. Also, please add the most
log file (ds4windows_log.txt) from the Logs folder in your DS4Windows config folder.

**Desktop (please complete the following information):**
 - Controller Make and Model: Sony DualSense
 - OS: Windows 10 21H1
 - DS4Windows Version 3.0.13

**Additional context**
One possible workaround is to have a permanent slot for X360 mode, however this only works up to the point where the PC goes into standby (shown at 3 AM in the logs), which causes DS4Windows to shut down its services and thus results in the same bug again.

Also I did test turning off DS4Windows and staying in DirectInput/XInput modes... HidHide does still work against PSO2NGS and hides the DualSense as a controller.

[ds4windows_log.txt](https://github.com/Ryochan7/DS4Windows/files/6912735/ds4windows_log.txt)

**Comment by Kanuan** [comment]
Maybe it's an issue with the XInput Slots order? See the discussion around here -> https://github.com/Ryochan7/DS4Windows/issues/2387#issuecomment-890534330

**Comment by velocity7** [comment]
I terminated all processes using XInput and still have the issue. See Process Explorer window:

![image](https://user-images.githubusercontent.com/11874183/127776925-4d38ae4d-3a98-497f-837f-f489a1541759.png)

**Comment by velocity7** [comment]
Just wanted to leave an update, it looks like if I uncheck the "Hide DS4 Controller" checkbox and just let HidHide do its thing, everything seems to return to normal? Will require more testing. I'll report back later if this doesn't pan out.

**Comment by Kanuan** [comment]
Let us know if the `Hide DS4 controllers` option is the culprit. It should be left disabled if already HidHide is being used to hide the controllers

**Comment by mika-n** [comment]
The best practice is to use either "Hide DS4 Controllers" (=exclusive mode) OR HidHide, but not both at the moment. These techniques are alternative ways to hide the physical gamepad:
https://github.com/Ryochan7/DS4Windows/wiki/Exclusive-Mode-(Hide-DS4-Controller-config-option)-tips-and-issues

**Comment by velocity7** [comment]
Unfortunately it doesn't look like that was the culprit...

**Comment by velocity7** [comment]
Hello, wanted to give a heads up. The problem occurred again, and previously restarting DS4Windows did not resolve the issue. However, a new version was available at the time, 3.0.15, which I then updated to... and miraculously it started working. Seems like the issue was resolved via one of the issues listed in 3.0.15, though I'm not sure which one. Haven't had any issues since then, though now I'm on version 3.0.16.
